rules/windows/discovery_system_time_discovery.toml

```
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2023/01/17"
+integration = ["windows"]
```

```suggestion
integration = ["windows", "endpoint"]
```

rules based on process creation events are compatible with `endpoint` integration too (Endpoint events)